examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1424 @@
+{
```

Comment:
<p>source and target -&gt; "source" and "target"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>